NitroxPatcher/Patches/Dynamic/SpawnEscapePodSupplies_OnNewBorn_Patch.cs

```diff
@@ -0,0 +1,23 @@
+namespace NitroxPatcher.Patches.Dynamic;
```

Comment:
I think this is already in master so might just need a quick rebase

Reply:
Will perform a rebase.